docs/resources/exit_network.md

```diff
@@ -0,0 +1,39 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_exit_network Resource - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  TODO: Exit Networks behave similarly to Remote Networks. For more information, see Twingate's documentation https://www.twingate.com/docs/exit-networks.
```

Comment:
@alexb-twingate please check which doc description should we set for ExitNetworks